.buildkite/filebeat/filebeat-pipeline.yml

```diff
@@ -111,64 +126,69 @@ steps:
           - github_commit_status:
               context: "filebeat: Module compat tests / previous minor"
 
-      - label: "Module compat tests: next major"
-        # Run module integration tests under next major of Elastic stack.
-        env:
-          STACK_ENVIRONMENT: "next-major"
-          TESTING_FILEBEAT_SKIP_DIFF: 1
+      - label: ":ubuntu: Filebeat Module Compatibility ITs (Previous Minor)"
         command: |
-          set -euo pipefail
```

Comment:
let's keep this? since we are running the changesets it maybe good to fail fast (via `set -eo pipefail`)